website/static/js/tests/registrationUtils.test.js

```diff
@@ -307,6 +309,13 @@ describe('Question', () => {
             assert.isTrue(q.isComplete());
         });
     });
+    describe('#isValid', () => {
+        it('is true if the Question\'s value is not empty', () => {
+            assert.isFalse(q.value.isValid());
+            q.value('not empty');
+            assert.isTrue(q.value.isValid());
+        });
+    });
     describe('#valid', () => {
```

Comment:
@barbour-em in that case can you remove all of the now unused custom validation logic from Question? And delete the associated tests?